azurerm/resource_arm_app_service_test.go

```diff
@@ -1255,6 +1281,37 @@ resource "azurerm_app_service" "test" {
 `, rInt, location, rInt, rInt)
 }
 
+func testAccAzureRMAppService_appCommandLine(rInt int, location string) string {
```

Comment:
since the schema's shared between the App Service and App Service Slot resources - could we add a test covering this being set for an App Service Slot too?